src/main/common/maths.c

```diff
@@ -111,7 +111,7 @@ int32_t applyDeadband(int32_t value, int32_t deadband)
     return value;
 }
 
-inline int constrain(int amt, int low, int high)
```

Comment:
Whats the reason of removal of the inline?

constrain code is being used quite a lot through the code